File: client/app/src/main/java/com/rh/heji/data/db/ErrorLogDao.java
package com.rh.heji.data.db;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.OnConflictStrategy;

/**
 * Date: 2021/3/2
 * Author: 锅得铁
 * #
 */
@Dao
public interface ErrorLogDao {

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    void install(ErrorLog errorLog);

}

File: client/app/src/main/java/com/rh/heji/data/db/ErrorLogDao.kt
package com.rh.heji.data.db

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.OnConflictStrategy

/**
 * Date: 2021/3/2
 * @author: 锅得铁
 * #
 */
@Dao
interface ErrorLogDao {
    @Insert(onConflict = OnConflictStrategy.REPLACE)
    fun install(errorLog: ErrorLog?)
}